lib/pensio_api/version.rb

```diff
@@ -1,3 +1,3 @@
 module PensioAPI
-  VERSION = '0.3.4'.freeze
+  VERSION = '0.3.5'.freeze
```

Comment:
This version is different from the one returned by getVersion - is that deliberate?

Reply:
Sir my humble apologies this was from some other project! I closed it no need to merge or anything at all. If you have any task for ruby in open source let me know I will sure contribute it as it would be great to learn from you. Thank you.